nemo/collections/nlp/modules/common/lm_utils.py

```diff
@@ -58,6 +58,7 @@ def get_lm_model(
     config_file: Optional[str] = None,
     checkpoint_file: Optional[str] = None,
     vocab_file: Optional[str] = None,
+    model_is_being_restored: Optional[bool] = False,
```

Comment:
I don't think we need this. `AppState` has the restore flag and you check for it there.

Reply:
https://github.com/NVIDIA/NeMo/blob/1d262c725df150ba837146e7899d8167adf524f6/nemo/utils/app_state.py#L381

Reply:
Agreed, I'd rather not pass this flag in every single method call, appstate is better for this inside the actual method in one place only.